src/tests/ftest/launch.py

```diff
@@ -1919,8 +1944,18 @@ def main():
     # Generate certificate files
     generate_certs()
 
+    # Determine the location of the avocado logs for archiving or renaming
+    data = get_output(["avocado", "config"]).strip()
+    avocado_logs_dir = re.findall(r"datadir\.paths\.logs_dir\s+(.*)", data)
+    avocado_logs_dir = os.path.expanduser(avocado_logs_dir[0])
+    print("Avocado logs stored in {}".format(avocado_logs_dir))
+
```

Comment:
I would remove this duplicated code (original is `run_test`) - see next comment.

Reply:
right, i meant to remove the dup inside of runtest but forgot. but if we move archive_cov into run_test, then will remove the one here